core/src/main/java/de/danoeh/antennapod/core/sync/SyncService.java

```diff
@@ -68,6 +70,7 @@
     private static final String TAG = "SyncService";
     private static final String WORK_ID_SYNC = "SyncServiceWorkId";
     private static final ReentrantLock lock = new ReentrantLock();
+    private static final EpisodeActionFilter episodeActionFilter = new EpisodeActionFilter();
```

Comment:
Looks like this can be a local variable instead of a static class variable. (There is already too much static stuff in AntennaPod that makes testing harder)

Reply:
These functions are static because they are pure functions. They don't make testing harder.
But having this as a class variable is unnecessary. I'll remove it